From c15eab6e3b35e9cbe0672c17cabbc14083247b92 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 29 May 2018 18:01:12 -0400
Subject: [PATCH] [scripts] tags.sh: Ignore redhat/rpm

Message-id: <20180529180112.28376-1-prarit@redhat.com>
Patchwork-id: 8101
O-Subject: [kernel team] [RHEL8.0 BZ 1582586] scripts/tags.sh: Ignore redhat/rpm
Bugzilla: 1582586
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

Bugzilla: http://bugzilla.redhat.com/1582586
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=16502319

RHEL_only.

If 'make cscope' is run on a tree which has had 'make rh-rpm',
'make rh-cross-*', etc., on it, the tags information contains entries
from the redhat/rpm/ directories.

Ignore the tags search in redhat/rpm.

Backport of RHEL7 464ad7779507 ("[scripts] tags.sh: ignore redhat/rpm").

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
Signed-off-by: Herton R. Krzesinski <herton@redhat.com>
---
 scripts/tags.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/tags.sh b/scripts/tags.sh
index 78e546f..805c73d 100755
--- a/scripts/tags.sh
+++ b/scripts/tags.sh
@@ -15,6 +15,8 @@ fi
 ignore="$(echo "$RCS_FIND_IGNORE" | sed 's|\\||g' )"
 # tags and cscope files should also ignore MODVERSION *.mod.c files
 ignore="$ignore ( -name *.mod.c ) -prune -o"
+# RHEL tags and cscope should also ignore redhat/rpm
+ignore="$ignore ( -path redhat/rpm ) -prune -o"
 
 # Do not use full path if we do not use O=.. builds
 # Use make O=. {tags|cscope}
-- 
2.9.0

